bedrock/firefox/templates/firefox/features/index.html

```diff
@@ -79,61 +79,73 @@ <h2 class="section-title"><span>{{ _('Modern') }}</span></h2>
         </p>
       </li>
       <li class="features-list-item">
-        <a href="{{ url('firefox.android.index') }}" data-link-type="link" data-link-name="Firefox for Android" data-link-position="modern 5">
+        <a href="{{ url('firefox.features.send-tabs') }}" data-link-type="link" data-link-name="Tabs That Travel" data-link-position="modern 5">
```

Comment:
This block should show up only if firefox/features/send-tabs is activated. With all the shared strings and included files, that's the only safe way to do it…